bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -265,6 +265,13 @@ public static void addMetadata(@NonNull final String tab,
         getClient().addMetadata(tab, key, value);
     }
 
+    /**
+     * Add metadata to subsequent exception reports with a Hashmap
+     */
+    public static void addMetadata(@NonNull final String tab, @NonNull final Map<String, ?> metadata) {
```

Comment:
Checkstyle isn't happy with how long this line is. 
```suggestion
    public static void addMetadata(@NonNull final String tab,
                                   @NonNull final Map<String, ?> metadata) {
```